bin/create.js

```diff
@@ -364,6 +449,22 @@ function parseDate (argv, field) {
     argv[field] = Math.floor(date.valueOf() / 1000);
 }
 
+function getSecret() {
+    try {
+        return (process.env.WEBTASK_JWT_SECRET || Crypto.randomBytes(128).toString('base64'));
```

Comment:
Does generation of a random key only really make sense with `--share`? Are we doing this validation somewhere?